ci-operator/step-registry/hypershift/azure/create/hypershift-azure-create-chain.yaml

```diff
@@ -54,10 +57,21 @@ chain:
         ENCRYPTION_KEY_ID="$(cat "${SHARED_DIR}/azure_active_key_url")"
       fi
       
-      /usr/bin/hypershift create cluster azure ${ICSP_COMMAND} \
+      EXTRA_ARGS=""
```

Comment:
Just to be curious, sounds like the content of the chian.yaml looks more like a ref step file, why here we are writing codes in a chain yaml file?